fw_gear_hcp_diff/args/DiffPreprocPipeline.py

```diff
@@ -0,0 +1,383 @@
+"""
+Builds, validates, and excecutes parameters for the HCP script
+/opt/HCP-Pipelines/DiffusionPreprocessing/DiffPreprocPipeline.sh
+"""
+
+import logging
+import os
+import os.path as op
+from collections import OrderedDict
+
+from tr import tr
+
+from utils.diff_utils import make_sym_link
+from ..gear_preliminaries import create_sanitized_filepath
+from .common import build_command_list, exec_command
+
+log = logging.getLogger(__name__)
+
+
+def build(context):
+    config = context.config
+    inputs = context._invocation["inputs"]
```

Comment:
Any reason to not use `context.inputs`?

Reply:
Old code... trying to leave virtually everything alone that I can, so there is less to have break from currently functional.